### DISCUSSION TITLE
Kingroon profiles missed on Linux version

### DISCUSSION THREAD

**Discussion by mechano** [OrcaSlicer/OrcaSlicer#1922] [General]
Good Morning.

Why the Linux version of v1.6.6 hasn't Kingroon KP3S Pro S1 and V2 printers?

They are on Windows version.

I opened the AppImage with a tool and saw that there are all the same profiles that are on Windows version, but when you run this AppImage very few printers are visible.

**Comment by mechano** [comment]
We are at 1.9.0 and still Kingroon and some other printer brands are present into the AppImage for Linux but not accessible when I try to add a printer.

**Reply by konradmb** [reply]
Should be fixed soon, when someone approves my PR: https://github.com/SoftFever/OrcaSlicer/pull/3712.
If you see other brands missing on Linux that are visible on Windows, it should be easy to fix that. In the meantime just rename the filenames in "vendor-name" directories to match with "vendor-name.json" profile. You can make a PR then for other brands.

**Comment by mechano** [comment]
Are you talking about the .config/OrcaSlicer directory or into the AppImage
file?

Because I don't know how to change into the AppImage and repack the file.



Il giorno mer 17 gen 2024 alle ore 10:09 konradmb ***@***.***>
ha scritto:

> Should be fixed soon, when someone approves my PR: #3712
> <https://github.com/SoftFever/OrcaSlicer/pull/3712>.
> If you see other brands missing on Linux that are visible on Windows, it
> should be easy to fix that. Just rename the filenames in "vendor-name"
> directories to match with "vendor-name.json" profile.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SoftFever/OrcaSlicer/discussions/1922#discussioncomment-8154027>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV5ZA2W5SGCRN5XPHIPKVDYO6ILNAVCNFSM6AAAAAA36YDX3OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCNJUGAZDO>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Reply by konradmb** [reply]
About the directory inside AppImage file.
You don't need to repack it (you can do it with https://github.com/AppImage/appimagetool if you really want, but that's not necessary). This is what I've been doing to test the changes:
1. Run `./OrcaSlicer_Linux_V1.9.0.AppImage --appimage-extract` in terminal to extract internal AppImage archive 
2. You should see a new `squashfs-root` directory inside the folder with AppImage
3. Go to `squashfs-root/resources/profiles` and change the filenames in `Kingroon/machine` folder like I did in https://github.com/SoftFever/OrcaSlicer/pull/3712/files
4. To run app use `AppRun` script in `squashfs-root` dir. You can right-click on it and select "Run as program" (at least on Gnome).

Now you can at least select the printers, but you won't see Kingroon filaments without other changes in https://github.com/SoftFever/OrcaSlicer/pull/3712/files

**Comment by konradmb** [comment]
There's a nightly build with included changes: https://github.com/SoftFever/OrcaSlicer/actions/runs/7558399292/artifacts/1175813848

**Comment by mechano** [comment]
Oh thanks, can't wait to try the new version.


Il giorno mer 17 gen 2024 alle ore 20:47 konradmb ***@***.***>
ha scritto:

> There's a nightly build with included changes:
> https://github.com/SoftFever/OrcaSlicer/actions/runs/7558399292/artifacts/1175813848
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SoftFever/OrcaSlicer/discussions/1922#discussioncomment-8161206>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAV5ZA6QU4RR5YDH3TZUPH3YPATDRAVCNFSM6AAAAAA36YDX3OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DCNRRGIYDM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Roberto Giuffrè
tel. +39-328-9140378
email: ***@***.***
pec: ***@***.***

**Comment by mechano** [comment]
1.9.1 is out and still lot of printers are not available in the Linux version. 
They are present into AppImage file but not selectable from UI.

**Reply by konradmb** [reply]
I guess the issue is the same thing: lowercase-uppercase mismatch. Someone needs to fix it. I pass, I don't have time (maybe some other day).
